includes/data-stores/class-wc-admin-reports-orders-data-store.php

```diff
@@ -279,6 +283,17 @@ public function get_data( $query_args ) {
 				return new WP_Error( 'woocommerce_reports_revenue_result_failed', __( 'Sorry, fetching revenue data failed.', 'wc-admin' ) );
 			}
 
+			$unique_products = $wpdb->get_var(
+				"SELECT
+						COUNT( DISTINCT {$wpdb->prefix}wc_order_product_lookup.product_id )
+					FROM
+						{$wpdb->prefix}wc_order_product_lookup JOIN wp_posts ON {$wpdb->prefix}wc_order_product_lookup.order_id = wp_posts.ID
+					WHERE
+						1=1
+						{$totals_query['where_clause']}"
+			); // WPCS: cache ok, DB call ok, unprepared SQL ok.
+			$totals[0]['products'] = $unique_products;
+
```

Comment:
Maybe it would be nicer to extract this whole new block as a method?

Reply:
Good idea. Moved this to a new block https://github.com/woocommerce/wc-admin/pull/613/commits/a219b9cf4ca57197176561c2fe928369356f3d5a